### DISCUSSION TITLE
Insert new row

### DISCUSSION THREAD

**Discussion by LoneWanderer92** [QtExcel/QXlsx#187] [Q&A]
Hi! I'm evaluating this library for handling some Excel reports, my goal is to open a template document (not a formal XLSXT file but a standard Excel file used as a template) and insert some data into it. 
So far so good I can update my cell without any problem and also setting the borders, but I can't find an API to insert a new row above or below a certain existing row; my template consist in a header and a footer and I'd like to add data between them without splitting the whole thing into multiple files or copy the footer in RAM.
The library Qxlnt have a nice 'insertRow' API, but I found more easy to use the QXlsx from the POV of Qt integration.

Does anybody knows how I can add a row? Is it a supported function?
Thanks and many compliments for the work on this library :D

**Comment by qlsbb999** [comment]
I have same question.

**Comment by j2doll** [comment]
Dear @LoneWanderer92 
Sorry for late reply.
The feature you mentioned seems to be unsupported (probably). 
Can you show me an Excel file for the feature you mentioned?
